tests/ReplayData/Repository.testGenerateReleaseNotes.txt

```diff
@@ -0,0 +1,10 @@
+https
+POST
+api.github.com
+None
+/repos/jacquev6/PyGithub/releases/generate-notes
```

Comment:
Are these test files hand crafted?
```suggestion
/repos/PyGithub/PyGithub/releases/generate-notes
```

Reply:
These files were generated with and then hand-edited.  I generated them using an OAuth token and then edited them to conform to the password-generated format.  Based on the previous testing setup, these were configured to use `jacquev6` in the test setup, so I edited the files to also use `jacquev6` so that the unit tests would pass.

This testing framework is novel to me, so there's a good chance I've done something wrong!

Here's my command for generating the ReplayData:

`pytest tests/Repository.py::Repository::testGenerateReleaseNotesWithAllArguments --record --auth_with_token`

I just now attempted to update `jacquev6` to `PyGithub` in the two newly test files, and I get this resulting error in the Repository test:

```
========================================================================= short test summary info ==========================================================================
FAILED Repository.py::Repository::testGenerateReleaseNotes - requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
FAILED Repository.py::Repository::testGenerateReleaseNotesWithAllArguments - requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
================================================================ 2 failed, 937 passed, 2 warnings in 30.15s ================================================================
```

I did a quick search for `jacquev6`, and found that this appears 100s of times in the test files, so it appears that these are pretty much expected to use `jacquev6` instead of the more appropriate `PyGitHub`.

So, in short, changing `jacquev6` to `PyGitHub` would be a much larger scope PR than this one (although it would probably be a really good cleanup!)